docs/cog_guides/mod.rst

```diff
@@ -0,0 +1,763 @@
+
+**Example Usage**
+
+* ``[p]kick 428675506947227648 wanted to be kicked.``
+    This will kick Twentysix from the server.
+* ``[p]kick @Twentysix wanted to be kicked.``
+    This will kick Twentysix from the server.
+
+If a reason is specified, it will be the reason that shows up
+in the audit log.
+
+.. _mod-command-massban:
+
+^^^^^^^
+massban
+^^^^^^^
+
+.. note:: |admin-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]massban <user_ids...> [days] [reason]
+
+.. tip:: Alias: ``hackban``
+
+**Description**
+
+Mass bans user(s) from the server.
+
+**Arguments**
+
+* ``<user_ids...>``: The users to ban. This must be a list of user IDs seperated by spaces.
+* ``[days]``: The amount of days of messages to cleanup on massban.
+* ``[reason]``: The reason why this user was banned.
+
+**Example Usage**
+
+* ``[p]massban 345628097929936898 57287406247743488 7 they broke all rules.``
+    This will ban all the added userids and delete 7 days of worth messages.
+
+.. _mod-command-modset:
+
+^^^^^^
+modset
+^^^^^^
+
+.. note:: |guildowner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]modset 
+
+**Description**
+
+Manage server administration settings.
+
+.. _mod-command-modset-defaultdays:
+
+""""""""""""""""""
+modset defaultdays
+""""""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]modset defaultdays [days=0]
+
+**Description**
+
+Set the default number of days worth of messages to be deleted when a user is banned.
+
+The number of days must be between 0 and 7.
+
+**Arguments**
+
+* ``[days=0]``: The default number of days of messages to be deleted when a user is banned.
+
+.. note:: This value must be between 0 and 7.
+
+.. _mod-command-modset-defaultduration:
+
+""""""""""""""""""""""
+modset defaultduration
+""""""""""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]modset defaultduration <duration>
+
+**Description**
+
+Set the default time to be used when a user is tempbanned.
+
+Accepts: seconds, minutes, hours, days, weeks
+
+**Arguments**
+
+* ``<duration>``: The default duration for when a user is temporarily banned. Accepts seconds, minutes, hours, days or weeks.
+
+**Example Usage**
+
+* ``[p]modset defaultduration 7d12h10m``
+* ``[p]modset defaultduration 7 days 12 hours 10 minutes``
+
+.. _mod-command-modset-deletenames:
+
+""""""""""""""""""
+modset deletenames
+""""""""""""""""""
+
+.. note:: |owner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]modset deletenames [confirmation=False]
+
+**Description**
+
+Delete all stored usernames and nicknames.
+
+**Arguments**
+
+- ``<confirmation>``: Whether to delete all stored usernames and nicknames. |bool-input|
+
+.. _mod-command-modset-deleterepeats:
+
+""""""""""""""""""""
+modset deleterepeats
+""""""""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]modset deleterepeats [repeats]
+
+**Description**
+
+Enable auto-deletion of repeated messages.
+
+**Arguments**
+
+* ``[repeats]``: The number of repeated messages to auto-delete.
```

Comment:
This should probably be changed to talk about how it's the number of repeated messages needed **before** further messages are deleted